data/human/transport missions.txt

```diff
@@ -3582,3 +3582,732 @@ mission "Hauler VI cargo prototype 3"
+			label notff
+			`	"Ready?" asks River.`
+			choice
+				`	"As I'll ever be."`
+			`	You are more prepared this time, but instead of the wind and cold of Hope, there is the heat and noise of the sweatshop. There is a large factory with its forth wall removed so you can see inside. There are hundreds of people sweating over dirty machines. Armed guards patrol and dole out beatings at random. Many are crying as they work. It's heart wrenching.`
+			` Your contribution to the project is front and center. A stream of slaves passes from the pirate to the Syndicate ship. It's much more visceral in life size than the little projector Bunker had. As a guard brings down a baton on a woman's head you find yourself trying to stop him, only to stumble through the hologram. Your heart is pounding.`
+			`	"I must have seen a hundred people do that just today," says Mariel. "Everyone either leaves crying or with their fists clenched. It definitely gets people emotional." Shes right. Even though you were out gunned and didn't have a choice, you still find yourself deeply regretting not trying to help these people. You see Ale and River are crying.`
+			`	"We are out of time. We need to reopen," says Mariel, pushing you toward the exit.`
+			`	"Wow, that's really intense," says Ale as everyone leaves. "We really deserve a vacation. Hey <first>, we have always wanted to go to Skymoot to see the dragons, do you think you could take us?"`
+			choice
+				`	"I would be happy to."`
+					goto happyto
+				`	"I should really be getting back to the war."`
+			`	"It's on the way but I can see you are busy. We will be in touch," she says, smiling.`
+				decline
+			label happyto
+			`	"Wonderful. We will get our stuff and meet you at the <ship>," she says, smiling.`
+				accept
+	on visit
+		dialog `You must wait for your escorts to reach this system.`
+	on complete
+		conversation
+			`As you bring the <ship> down to the spaceport you see a dragon flying nearby and point it out to your passengers who 'ooo' and 'ahh'. They can hardly contain their excitement as they leave the ship. Each one gives you a hug and wishes you good luck.`
+			`	"We will definitely be in touch," says Ale as they leave.`
+
+
+
+mission "Freedom Uprising 1"
+	landing
+	deadline 14
+	source
+		government "Syndicate"
+	destination "Moonshake"
+	to offer
+		has "Smuggler's Den, Part 2: done"
+		has "Journalism 5: done"
+	on offer
+		conversation
+			`	Back on your ship you notice a new message from Ale: "Hey. We are nearly done with the Syndicate Slavery project but it will still be a while for the exposition is ready to tour. In the meantime we have another project, but as usual it's time sensitive. Meet up with us on Moonshake soon or we will leave without you. Same place as before."`
```

Comment:
```suggestion
			`	Back on your ship you notice a new message from Ale: "Hey. We are nearly done with the Syndicate Slavery project but it will still be a while before the exposition is ready to tour. In the meantime we have another project, but as usual it's time sensitive. Meet up with us on Moonshake soon or we will leave without you. Same place as before."`
```